.env.sample

```diff
@@ -5,12 +5,9 @@ BOOTNODE_NODE_ID_LOCAL=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
 CHAIN_VERSION=dev
 ENDPOINT_DEVELOPMENT=127.0.0.1
 ENDPOINT_TESTNET=node-1
+
 # Development or Production
 NODE_ENV=Development
 NODE_KEY=88dc3417d5058ec4b4503e0c12ea1a0a89be200fe98922423d4334014fa6b0ee
-BOOTNODE_NODE_ID_TESTNET=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
```

Comment:
i proposed changing it from `BOOTNODE_ID...` TO `BOOTNODE_NODE_ID` so it's clear we're talking about a Node ID

Reply:
we only define BOOTNODE_ID OR BOOTNODE_KEY (fake one in source control), its pretty clear I think

Reply:
ok maybe rename `NODE_KEY` to `BOOTNODE_KEY` for consistency?

Reply:
and change `BOOTNODE_NODE_ID_LOCAL` to `BOOTNODE_ID_LOCAL`